source/FAST/Visualization/WindowWidget.cpp

```diff
@@ -17,6 +20,28 @@ void WindowWidget::keyPressEvent(QKeyEvent *event) {
         view->keyPressEvent(event);
 }
 
+void WindowWidget::dragEnterEvent(QDragEnterEvent *event) {
```

Comment:
Why is the dragEnterEvent needed? Is it not enough with just dropEvent?

Reply:
I think it is because I use QMimeData for storing the filenames (urls) in a container before the drop event. I followed the example from here, and seemed to make sense at the time:
https://doc.qt.io/qt-5/qmimedata.html

If there is a smarter way to do it, I would be happy to know.